addons/hr_timesheet/tests/test_timesheet.py

```diff
@@ -14,6 +14,7 @@ class TestCommonTimesheet(TransactionCase):
     def setUpClass(cls):
         super(TestCommonTimesheet, cls).setUpClass()
 
+        create_date = Datetime.now()
```

Comment:
Why do you set a create_date? It will be the same if you don't provided it no? Isn't it taking the same value?